backend/api/Controllers/UserController.cs

```diff
@@ -120,14 +120,10 @@ public IActionResult GetAccessRequest(int id)
         [SwaggerOperation(Tags = new[] { "user" })]
         public async Task<IActionResult> AddAccessRequestAsync([FromBody] Model.AccessRequestModel model)
         {
-            if (model == null || model.Agencies == null || model.Roles == null)
+            if (model == null || model.Roles == null)
             {
                 throw new BadRequestException("Invalid access request specified");
             }
-            if (model.Agencies.Count() != 1)
-            {
-                throw new BadRequestException("Each access request can only contain one agency.");
-            }
```

Comment:
We no longer have a need for this check. The frontend will always send null agencies